### ISSUE TITLE
Difference between code and paper

### ISSUE THREAD

**Issue by valencebond** [tim-learn/SHOT#6]
thanks for your code and paper, the proposed new problem is interesting. However, there are some inconsistencies between code and paper.
1. according to the paper, the total loss consists of three parts, a entropy loss Lent, a diversity loss Ldiv and a minus cross entropy loss. In the code, classifier_loss corresponds to minus [cross entropy loss](https://github.com/tim-learn/SHOT/blob/5a00a47947d96233c0e45157b2fcec77fe48b869/object/image_target.py#L193)， entropy_loss corresponds to [Lent](https://github.com/tim-learn/SHOT/blob/5a00a47947d96233c0e45157b2fcec77fe48b869/object/image_target.py#L201), gentropy_loss corresponds to [Ldiv](https://github.com/tim-learn/SHOT/blob/5a00a47947d96233c0e45157b2fcec77fe48b869/object/image_target.py#L204)

2. As paper said, cross entropy loss is negative and Ldiv is positive, but cross entropy loss is positive and Ldiv is negative in code.

please correct me if there is something wrong.

**Comment by tim-learn** [comment]
I don't know that our paper have said "cross entropy loss is negative and Ldiv is positive". I hope you can check what is KL loss and what is cross-entropy loss carefully, especially the existence of minus symbol.
